client/packages/system/src/Patient/VaccinationCards/DetailView/DetailView.tsx

```diff
@@ -1,63 +1,24 @@
 import React, { FC, useEffect } from 'react';
 import {
-  TableProvider,
-  createTableStore,
   useBreadcrumbs,
-  createQueryParamsStore,
   useParams,
-  InlineSpinner,
   useTranslation,
-  DataTable,
-  useColumns,
-  NothingHere,
   useIntlUtils,
-  useFormatDateTime,
-  useRowStyle,
-  useTheme,
-  StatusCell,
+  InlineSpinner,
 } from '@openmsupply-client/common';
 import { usePatientVaccineCard } from '../../api/hooks/usePatientVaccineCard';
-import { VaccinationCardItemFragment } from '../../api/operations.generated';
-
-const useStyleRowsByStatus = (rows?: VaccinationCardItemFragment[]) => {
-  const { setRowStyles } = useRowStyle();
-  const theme = useTheme();
-
-  useEffect(() => {
-    if (!rows) return;
-
-    const doneRows = rows.filter(row => row.given).map(row => row.id);
-    const notDoneRows = rows.filter(row => !row.given).map(row => row.id);
+import { VaccinationsTab } from 'packages/system/src/Encounter/DetailView/VaccinationsTab';
 
-    setRowStyles(doneRows, {
-      backgroundColor: `${theme.palette.background.success} !important`,
-    });
-    setRowStyles(
-      notDoneRows,
-      {
-        backgroundColor: 'white !important',
-      },
-      // Parameter to prevent the previous setRowStyles from being
-      // reset/overwritten
-      false
-    );
-  }, [rows]);
-};
-
-export const VaccinationCardComponent: FC = () => {
+export const VaccinationCardDetailView: FC = () => {
   const t = useTranslation('dispensary');
   const { programEnrolmentId = '' } = useParams();
   const { setCustomBreadcrumbs } = useBreadcrumbs();
-  const { localisedDate } = useFormatDateTime();
   const { getLocalisedFullName } = useIntlUtils();
-  const theme = useTheme();
 
   const {
     query: { data, isLoading },
   } = usePatientVaccineCard(programEnrolmentId);
```

Comment:
I don't like calling this hook in here as well as in the child "VaccineCardTable" component, but couldn't see any other straightforward way to access the patient data for the breadcrumbs. Presumably the caching will prevent duplicate network request?

Reply:
Ah yeah, oh well. Shooould get the caching? But not too worried, should only need to run once here aynway. And I suppose the same as running a separate query just for this data 🤷‍♀️